### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by czystyl**
## Summary:
- Added examples how schema looks like without `@rest`directive
- Added tests for `schemaWithResolvers`
- Updated readme

### CODE REVIEW

**Comment by czystyl** [comment]
I added CircleCi but not runs on this PR

File: README.md [hunk 1]
```diff
@@ -19,52 +19,124 @@ Using this directive allows you to make REST integrations without creating any r
...
+module.exports = makeExecutableSchema({
+  typeDefs,
+  schemaDirectives: {
+    rest: restDirective,
+  },
+});
 ```
 
-GraphQL queries:
+> Warning! Directive overwrites your resolvers if defined.
```
Comment (jukben) [inline README.md]: typo: `Warning! Directive overwrites your resolvers if they're defined.`

File: example/schema.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 const { makeExecutableSchema } = require('graphql-tools');
 const restDirective = require('../index');
 
+const GITHUB_URL = 'https://api.github.com';
+const USER_URL = 'https://randomuser.me/api';
+const ADMIN_URL = 'https://yesno.wtf';
```
Comment (jukben) [inline example/schema.js]: nip: I would go for `https://yesno.wtf/api` just for the convention.

File: README.md (hunk 1 above)
Comment (czystyl) [reply README.md]: Me too 😞 :D

File: __tests__/schema.test.js [hunk 3]
```diff
@@ -45,12 +45,12 @@ test('path is return correct result and fillParametersFromArgs replace parameter
     {},
     {},
     { gender: 'female' }
-  );
-
-  expect(response).toMatchSnapshot();
+  ).then(response => {
+    expect(response).toMatchSnapshot();
```
Comment (thymikee) [inline __tests__/schema.test.js]: Why not async/await?

Comment (czystyl) [reply __tests__/schema.test.js]: Tests fail on node 6. If we want to add async/await then we need to add babel. 

Would you like to use babel?

Comment (thymikee) [reply __tests__/schema.test.js]: Meh, don't care. Anyway, you can always add a single plugin and use it just in test environment

Comment (czystyl) [reply __tests__/schema.test.js]: For now, I removed babel from the project for simplicity.

Comment (thymikee) [reply __tests__/schema.test.js]: cool, I'm good with that 👍

**Pull request merged**